CMakePresets.json

```diff
@@ -0,0 +1,117 @@
+{
+    "version": 3,
+    "cmakeMinimumRequired": {
+        "major": 3,
+        "minor": 20,
+        "patch": 0
+    },
+    "configurePresets": [
+        {
+            "name": "freerdp",
+            "displayName": "freerdp library",
+            "cacheVariables": {
+				"WITH_FFMPEG": "OFF",
+				"WITH_SWSCALE": "OFF",
+				"CHANNEL_URBDRC": "OFF",
+				"WITH_CLIENT": "OFF"
```

Comment:
you´re building `WITH_SERVER=ON` and `WITH_PROXY=ON`
they might actually be good things to consider for options ;)

Reply:
Service is working now?

Reply:
> Service is working now?

what do you mean by that?

Reply:
I remember when I used it last year, you said it was unfinished.

Reply:
WITH_SERVER=ON include server library and server application. so that whether it needs to be add WITH_SERVER_APP?
The WITH_CLIENT too.